src/org/loklak/harvester/WeiboInfoScraper.java

```diff
@@ -0,0 +1,31 @@
+package org.loklak.harvester;
+import java.util.regex.Pattern;
+
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.*;
+import org.jsoup.select.Elements;
+import java.util.regex.Matcher;
+
+public class WeiboInfoScraper {
+	/**
+	 * This is a helper function that helps user to extract html nested inside of html script
+	 * @param raw_html
+	 * @return nested html String
+	 */
+	public static String getNestedHtml(String raw_html){
+		String html = raw_html.replace("\\","");
```

Comment:
Can we pass the URL directly to ```getNestedHtml``` , call the ```buildRequest``` and get the html ? So that i can just use getNestedHtml method directly. 
Also what folder structure do you suggest? Can we create a weibo folder and add all the request files into it?